packages/ui/react-ui-editor/src/extensions/comments.ts

```diff
@@ -327,16 +327,8 @@ export const createComment: Command = (view) => {
   const cursor = Cursor.getCursorFromRange(view.state, { from, to });
   if (cursor) {
     // Create thread via callback.
-    const id = options.onCreate?.({ cursor, from, location: view.coordsAtPos(from) });
-    if (id) {
-      // Update range.
-      view.dispatch({
-        effects: setSelection.of({ current: id }),
-        selection: { anchor: to },
-      });
-
-      return true;
-    }
```

Comment:
This selection gets set in the bi-directional linking based on `scroll_into_view`